apps/web/src/ee/clerk/components/UserProfileButton.tsx

```diff
@@ -1,29 +1,24 @@
 import { UserButton } from '@clerk/clerk-react';
 import { IconBolt } from '@novu/novui/icons';
-import { FeatureFlagsKeysEnum } from '@novu/shared';
 import { ROUTES } from '../../../constants/routes';
 import { useNewDashboardOptIn } from '../../../hooks/useNewDashboardOptIn';
-import { useFeatureFlag } from '../../../hooks';
 import { WEB_APP_URL } from '../../../config';
 
 export function UserProfileButton() {
   const { optIn } = useNewDashboardOptIn();
-  const isNewDashboardEnabled = useFeatureFlag(FeatureFlagsKeysEnum.IS_NEW_DASHBOARD_ENABLED);
 
   return (
     <UserButton
       afterSignOutUrl={`${WEB_APP_URL}${ROUTES.AUTH_LOGIN}`}
       userProfileUrl={ROUTES.MANAGE_ACCOUNT_USER_PROFILE}
     >
-      {isNewDashboardEnabled && (
```

Comment:
This will also show this to v1 users right? I'm not sure how their experience will be 🤔

Reply:
They should see the button, go to the new Dashboard, see all their workflows but for V1 workflows go back to the old web app on workflow selection.

I will verify it on staging.